src/components/FormMetadataEditor.jsx

```diff
@@ -40,6 +42,15 @@ const BasicFormMetadataEditor = ({ form, onChange, usageContext }) => {
         readOnly={usageContext === "edit"}
         onChange={(event) => onChange({ ...form, name: event.target.value })}
       />
+      <Input
+        label="Tema"
+        type="text"
+        id="tema"
+        placeholder="Skriv inn tema"
```

Comment:
Endre til "Skriv inn temakode (OPP)" for å tydeliggjøre hvilken input som forventes